gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/FormatStep.java

```diff
@@ -168,6 +169,19 @@ public Set<String> getScopeKeys() {
         return variables;
     }
 
+    @Override
+    public Set<ScopingInfo> getScopingInfo() {
```

Comment:
As each step contains a Set of labels, but only one Pop, would it be easier for ScopingInfo to be defined as
```
public static class ScopingInfo {
    public Set<String> labels;
    public Pop pop;
```
and for this method to return `ScopingInfo` instead of `Set<ScopingInfo>`